Code/EnginePlugins/JoltPlugin/Actors/JoltDynamicActorComponent.h

```diff
@@ -83,6 +83,9 @@ class EZ_JOLTPLUGIN_DLL ezJoltDynamicActorComponent : public ezJoltActorComponen
   /// \brief Should be called when a constraint is removed (though not strictly required) to prevent unnecessary message sending.
   void RemoveConstraint(ezComponentHandle hComponent);
 
+  /// \brief Returns the actual mass of this actor which is either user defined or calculated from density.
```

Comment:
It should mention that this is also dependent on the runtime state (kinematic vs dynamic) and my differ from the configured mass.